### PR TITLE
Update documention for tutorial

### PR DESCRIPTION
**Pull request by Billy99**
With all the API changes, update the documentation around the tutorial.

### CODE REVIEW

File: docs/logging.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+Aug 08 16:25:04 ebpf03 bpfd[180118]: Loaded static program pass with UUID a3ffa14a-786d-48ad-b0cd-a4802f0f10b6
+```
+
+Stop the service:
+
+```console
+sudo systemctl stop bpfd.service
+```
+
+## Kubernetes Deployment
```
[the remark this answers is not in the document]
Comment (astoycos) [reply docs/logging.md:92]: Maybe this actually goes in the dev docs too

**Review by astoycos — COMMENTED** [review]
Really just some Nits

File: docs/logging.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Billy99) [reply docs/logging.md:92]: Done. Depending on how you lay out the new website headings, the logging.md should go under any developer type heading.

**Review by astoycos — APPROVED** [review]
LGTM

Huge thanks @Billy99

**Pull request merged**